src/language-css/parser-postcss.js

```diff
@@ -443,6 +448,7 @@ function parseNestedCSS(node, options) {
         }
       }
     }
+    console.log(node.type);
 
```

Comment:
```suggestion
if (node.type === "value-func" && node.value === "selector") {
```

something like  this. Will this  work?

Reply:
No, node.type is never value-func here.

But I have found another way, will update in a sec.